pkg/k8s/watchers/cilium_endpoint.go

```diff
@@ -108,7 +111,19 @@ func (k *K8sWatcher) endpointUpdated(oldEndpoint, endpoint *types.CiliumEndpoint
 			k.policyManager.TriggerPolicyUpdates(true, "Named ports added or updated")
 		}
 	}()
-
+	// handle endpoint_propagation_delay_seconds Metric for CEP create
+	if oldEndpoint == nil && endpoint != nil {
+		if p := k.endpointManager.LookupPodName(k8sUtils.GetObjNamespaceName(endpoint)); p != nil {
+			timeSinceCepCreated := time.Since(p.GetCreatedAt())
+			metrics.EndpointPropagationDelay.WithLabelValues().Observe(timeSinceCepCreated.Seconds())
+		}
+		// handle endpoint_propagation_delay_seconds Metric for CEP update
```

Comment:
> I removed myself from the reviewers list because of lack of cycles, but I see that Tam has been reviewing the changes. @zhanghe9702 Thanks for the PR. Do you need any help in moving it forward?

sry, in fact, i had try to do some modify, but i thought it's not include all update endpoint cases, and i add an identity-->timestamp map for record the update timestamp, if anyone have time to help me , i think i coud continue to solve this issue.